src/drive.ts

```diff
@@ -74,7 +74,13 @@ export class CS3Contents implements Contents.IDrive {
     const activeTab: string = (await this._state.fetch('activeTab')) as string;
 
```

Comment:
We agreed that we would not use drive names in the frontend anymore, so why are we adding it again? Do we really need it? Wouldn't we be able to detect from the path instead of having to append drives and then check them in the backend to do a replace?

Reply:
Yes, unfortunately we need it to make it work. The reason is in order to hide breadcrumbs I have to send incomplete file path in dir model to frontend, when the user clicks on a file, the request going to backend contains that incomplete path and without the name of the drive included it is impossible to determine what prefix should be added